install.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+declare -A deps
+deps["lateralgm.jar"]="IsmAvatar/LateralGM"
+deps["plugins/enigma.jar"]="enigma-dev/lgmplugin"
+
+get_latest () {
+  local release=$(curl --silent "https://api.github.com/repos/$1/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
You can use `grep --only-matching` to avoid piping to sed after grep (and probably make your match easier to read).